### ISSUE TITLE
Quest Preview 

### ISSUE THREAD

**Issue by Hudsonplayzy** [FTBTeam/FTB-App#1047]
### What is your suggestion?

I personally think it would be very neat if there were a feature to preview quests on a mod pack that has mods that allow questing. This would make it easier to find your favorite modpack.

### Alternatives ideas

A quest preview in the form of text under a description in a mod pack.

**Comment by CoryNB** [comment]
are you referring to oculus quest VR headsets for modpack support? there is a way to play them as certain VR must be used to play the game with a VR headset.

**Comment by Hudsonplayzy** [comment]
I mean quests as in ftb quests like the objectives in the modpacks.

**Comment by MichaelHillcox** [comment]
This is an interesting feature request, I'll note it down for review later on down the line :+1:

**Comment by CoryNB** [comment]
are you refering to vr minecraft? that is possible with modpacks just with a headset, excluding vision pro due to it being apple only and not compatible with pc vr mode.

**Comment by MichaelHillcox** [comment]
They are not talking about VR please stop asking

**Comment by MichaelHillcox** [comment]
This is a cool idea but due to how complex the quests data is, requiring item loading & rendering from game files, then requiring custom loading of `SBNT` files, etc. I think this falls too far out of scope for the app.

Sorry for the long wait on the rejection here.
